cli/azd/cmd/init.go

```diff
@@ -72,6 +73,12 @@ func (i *initFlags) Bind(local *pflag.FlagSet, global *internal.GlobalCommandOpt
 		"b",
 		"",
 		"The template branch to initialize from. Must be used with a template argument (--template or -t).")
+	local.StringSliceVar(
+		&i.templateTags,
+		"tags",
```

Comment:
@kristenwomack @wbreza , did you maybe considered `filter`  or `filter-tags` ?

Without reading at the help, the flag by itself is a little confusing (to me, at least).  I think I am totally bias by the `tag` flag from Docker cli and the git tag command, but, reading `tags` makes me thing on an action-flag that will create/write something.   I can't help thinking `azd init --tags foo` will add a tag `foo` to my project

Reply:
I'm open to suggestions on argument naming.

Reply:
I like `filter` or `filter-tags` a lot. It immediately makes my brain think of reducing a list to something specific.

Reply:
Updated to `--filter, -f`.  Today we only support flags but in the future we could expand to filter on other options so keeping this more generic to `filter` feels like the way to go in my opinion.